tsconfig.json

```diff
@@ -3,9 +3,9 @@
     "esModuleInterop": true,
     "noImplicitAny": false,
     "strictNullChecks": false,
-    "target": "es5",
-    "module": "commonjs",
-    "moduleResolution": "node",
+    "target": "ESNext",
```

Comment:
This looks pretty good, but why change our target here? If we're relying on TS for transpilation, don't we want to maintain an ES5 target for browser compatibility?

Reply:
I thought that the target could be part of the issue, but things didn't actually get fixed until I introduced Vite. That's a good call-out. I'll try switching these back and test to make sure that the error still gets fixed.